spec/helpers/webmock_helpers.rb

```diff
@@ -122,6 +122,24 @@ def contacts_find_query_stub(endpoint, query, success = true)
     end
   end
 
+  def contacts_find_all_stub(endpoint, ids, success = true)
+    if success
+      stub_request(
+        :get,
+        "#{endpoint}/private/api/v2/json/contacts/list?#{ids.to_query('id')}")
+        .to_return(
+          body: File.read('./spec/fixtures/contact_find_query.json'),
+          headers: { 'Content-Type' => 'application/json' },
+          status: 200
+        )
+    else
+      stub_request(
+        :get,
+        "#{endpoint}/private/api/v2/json/contacts/list?#{ids.to_query('id')}")
+        .to_return(status: 204)
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.